initializr-web/src/main/resources/static/js/auth.js

```diff
@@ -0,0 +1,43 @@
+(function () {
+
+    // Enter Global Config Values & Instantiate ADAL AuthenticationContext
+    window.config = {
+        instance: 'https://login.partner.microsoftonline.cn/',
```

Comment:
After having a determined FQDN, we will need to register a multi-tenant app to allow anyone sign in. 
https://docs.microsoft.com/en-us/azure/active-directory/develop/active-directory-devhowto-multi-tenant-overview

Reply:
Create an issue to track this? Or we do this now?